kitsune/questions/config.py

```diff
@@ -34,37 +34,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "Install and Update",
                             {
-                                "name": _lazy("Download, install and migration"),
-                                "topic": "download-and-install",
-                                "tags": ["download-and-install"],
+                                "name": _lazy("Install and Update"),
+                                "topic": "install-and-update",
+                                "tags": ["install-and-update"],
                             },
                         ),
                         (
-                            "privacy-and-security",
+                            "protect-your-privacy",
                             {
                                 "name": _lazy("Privacy and security settings"),
-                                "topic": "privacy-and-security",
-                                "tags": ["privacy-and-security"],
+                                "topic": "protect-your-privacy",
+                                "tags": ["protect-your-privacy"],
                             },
                         ),
                         (
                             "customize",
```

Comment:
Does this slug have to match the topic slug? In other words, should it be `customize-settings-and-preferences` instead of `customize`, or does that not matter?

Reply:
It doesn't matter what's the value there